types/roslib/tslint.json

```diff
@@ -10,6 +10,7 @@
         "prefer-const": false,
         "prefer-template": false,
         "space-before-function-paren": false,
-        "strict-export-declare-modifiers": false
+        "strict-export-declare-modifiers": false,
+        "unified-signatures": false
```

Comment:
Rather than ignore this rule wholesale for the package, it's probably more advisable to ignore the rule at the specific signature that has the problem (with some sort of comment explaining why).

Largely, it's best for this file to not contain any overrides, but currently it contains a lot...

Reply:
> Rather than ignore this rule wholesale for the package, it's probably more advisable to ignore the rule at the specific signature that has the problem (with some sort of comment explaining why).

Hi @jakebailey, understood. I have incorporated your suggestion in the latest commit. Thank you for the suggestion!